### PR TITLE
refactor: Update examples wrt 242

### PR DESCRIPTION
**Pull request by prmukherj**
Move examples from  [pyfluent-examples](https://github.com/ansys/pyfluent-examples) repo to the pyfluent core repo. Update the example scripts with settings-api usage.

### CODE REVIEW

File: examples/00-fluent/modeling_ablation.py [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+####################################################################################
+# .. math::
+#
+#    r = A \cdot p^n
+
+
+# %%
+# .. image:: ../../_static/ablation-problem-schematic.png
+#    :align: center
+#    :alt: Problem Schematic
```
Comment (hpohekar) [inline examples/00-fluent/modeling_ablation.py:48]: @prmukherj  Images associated with each example script are missing. e.g. `ablation-problem-schematic.png`. All of them should be uploaded into `_static` directory.

**Comment by hpohekar** [comment]
> Move examples from example-data repo to the pyfluent core repo. Update the example scripts with settings-api usage.

@prmukherj We are moving examples from [pyfluent-examples](https://github.com/ansys/pyfluent-examples) repo not example-data.

File: examples/00-fluent/modeling_ablation.py (hunk 1 above)
Comment (prmukherj) [reply examples/00-fluent/modeling_ablation.py:48]: Done, Thank you.

File: examples/00-fluent/conjugate_heat_transfer.py [hunk 2]
```diff
@@ -0,0 +1,874 @@
...
+    }
+)
+
+meshing.workflow.TaskObject["Describe Geometry"].Execute()
+
+#############################################################################
+# Update Interface Boundaries; Create Region
+# ==========================================
+
+meshing.workflow.TaskObject["Update Boundaries"].Arguments.setState(
```
Comment (seanpearsonuk) [inline examples/00-fluent/conjugate_heat_transfer.py:113]: @prmukherj this is one of the interfaces we'd like to improve for enhance meshing

File: examples/00-fluent/conjugate_heat_transfer.py [hunk 3]
```diff
@@ -0,0 +1,874 @@
...
+    CommandName=r"RunCustomJournal"
+)
+meshing.workflow.TaskObject["Run Custom Journal"].Rename(NewName=r"set-periodicity")
+meshing.workflow.TaskObject["set-periodicity"].Arguments = dict(
+    {
+        r"JournalString": r"""/bo rps translational semi-auto periodic-1-high periodic-2-high periodic-3-high periodic-4-high , 0 0 -2.3  # noqa: E501
+/bo rps translational semi-auto periodic-5* , 0 0 -2.3
+/bo rps translational semi-auto periodic-6-high , 0 0 -2.3
+/bo rps translational semi-auto periodic-7-high , 0 0 -2.3
+""",
```
Comment (seanpearsonuk) [inline examples/00-fluent/conjugate_heat_transfer.py:338]: There needs to be better API coverage for meshing.

File: examples/00-fluent/conjugate_heat_transfer.py (hunk 2 above)
Comment (prmukherj) [reply examples/00-fluent/conjugate_heat_transfer.py:113]: yes, agreed.

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**